git_machete/utils.py

```diff
@@ -319,3 +319,9 @@ class SyncToParentStatus(Enum):
     SyncToParentStatus.InSyncButForkPointOff: "?-",
     SyncToParentStatus.OutOfSync: "x-"
 }
+
+
+class CommandOutput(NamedTuple):
```

Comment:
```suggestion
class CommandResult(NamedTuple):
```
As `exit_code` can hardly be considered a part of "output"